src/CursorHandler.ts

```diff
@@ -0,0 +1,75 @@
+import { MissingCursorChange } from "./errors";
```

Comment:
can you use lowercase the files and use dashes for separation? That's the current convention we use in @octokit, with the exception of https://github.com/octokit/types.ts/tree/master/src